.gitignore

```diff
@@ -7,4 +7,6 @@ node_modules
 !Navigation*/src/node_modules
 !Navigation*/test/node_modules
 .DS_Store
-
+.project
+org.eclipse.buildship.core.prefs
+.classpath
```

Comment:
I'm reluctant to take these changes. Could you explain how they help you, please?